src/main/java/org/sonarlint/intellij/core/SonarLintServerManager.java

```diff
@@ -140,6 +142,9 @@ public void run() {
   }
 
   private SonarLintFacade createConnectedFacade(Project project, String serverId, String projectKey) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "createConnectedFacade" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)